CHANGES

```diff
@@ -1,12 +1,16 @@
 Revision history for Shaderc
 
-v2019.1-dev 2019-06-04
+v2019.12 2020-01-22
  - glslc, libshaderc:
    - Add -fnan-clamp: Generate code for max and min builtins so that,
      given a NaN operand, will return the other operand. Similarly, the
      clamp builtin favours non-NaN operands, as if clamp was implemented
      as the composition of max and min.
    - Add -finvert-y
+   - Using SetBeforeHlslLegalization for more relaxed validation rules
+   - Added support for SPIR-V 1.5
+   - Add --emit-line-derective option
```

Comment:
"directive"

Reply:
Hm.. I don't see this in glslc --help.   (Can fix later)

Reply:
Done